Sources/ServiceDiscovery/ServiceDiscovery.swift

```diff
@@ -12,179 +12,26 @@
 //
 //===----------------------------------------------------------------------===//
 
-import Atomics
-import Dispatch
-
 // MARK: - Service discovery protocol
 
 /// Provides service instances lookup.
 ///
 /// ### Threading
 ///
 /// `ServiceDiscovery` implementations **MUST be thread-safe**.
-public protocol ServiceDiscovery: AnyObject {
-    /// Service identity type
-    associatedtype Service: Hashable
-    /// Service instance type
-    associatedtype Instance: Hashable
-
-    /// Default timeout for lookup.
-    var defaultLookupTimeout: DispatchTimeInterval { get }
+public protocol ServiceDiscovery: Sendable {
```

Comment:
Protocol has the same name as module. ‼️🚨 For using namespaces? Module rename to `ServiceDiscoveryModule`?

Reply:
Or maybe rename one of them to just `Discovery`?

Reply:
> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).

So ServiceDiscovering?